content/guides/getting-started/component-framework-configuration.md

```diff
@@ -401,17 +401,15 @@ const { getWebpackConfig } = require('nuxt')
         webpackConfig,
       })
     },
-    componentFolder: 'components',
-    testFiles: '**/*.spec.js'
+    specPattern: 'components/**/*.cy.{js,jsx,ts,tsx}'
```

Comment:
Like with Next.js, I think for Nuxt, we'll want to use a `specPattern` like `'{pages,components}/**/*.cy.{js,jsx,ts,tsx}'`.